src/main/java/ai/elimu/web/project/app_category/AppCategoryDeleteController.java

```diff
@@ -20,23 +18,17 @@
 private final Logger logger = Logger.getLogger(getClass());
     
     @Autowired
-    private ProjectDao projectDao;
-    
-    @Autowired
     private AppCategoryDao appCategoryDao;
     
-    @RequestMapping(value = "/{id}", method = RequestMethod.GET)
+	@RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
```

Comment:
By the way, how will you make the request? With a normal link, or Ajax request? If using a ling, will HTTP DELETE work?

Reply:
DELETE request would have to be done with an ajax call, I presume. I am looking into this now. A GET is maybe simpler, on the other hand?
I will Try to do it with a DELETE, to make it as by REST conventions

Reply:
Ok. So since the way to solve this with least amount of code would be with a normal URL, we should probably just use `GET`, right? So please ignore my initial comment :-)

Reply:
> A GET is maybe simpler, on the other hand?

Yes, using `GET` would probably require less code. So let us go for the original solution?

Reply:
Ok, noted, I will change when I commit my next changes